### PR TITLE
extract style into external css

### PR DESCRIPTION
**Pull request by simonamarcu2**

### CODE REVIEW

File: web/simonamarcu2/html/primul/simy.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  font-size: larger;
+}
+
+h1 {
+  background-color: aquamarine;text-align: center; border: 4px solid violet font family cursive ;
+}
+
+h2 {
+  background-color: hsl(125, 54%, 73%);text-align: center; font-family: cursive;
+}
```
Comment (RalucaRbc) [inline web/simonamarcu2/html/primul/simy.css:17]: You need an empty line between the style of the divs

File: web/simonamarcu2/html/primul/simy.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
+body {
+  background-color: lavender;
+  font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
+  font-size: larger;
+}
+
+h1 {
+  background-color: aquamarine;text-align: center; border: 4px solid violet font family cursive ;
```
Comment (RalucaRbc) [inline web/simonamarcu2/html/primul/simy.css]: Each attribute on a different line

**Review by RalucaRbc — CHANGES_REQUESTED** [review]
You need to change few lines

**Review by RalucaRbc — APPROVED** [review]
💯

**Review by vasilemaftita — APPROVED** [review]
Nice: Work.

File: web/simonamarcu2/html/primul/index.html [hunk 3]
```diff
@@ -1,41 +1,68 @@
...
+      <cite>Young Living </cite> was founded in 1993, when Gary Young and Mary
+      Young developed their first organic herb farming distilation operation.
+    </p>
+    <!-- remember to add more info here -->
+    <p>
+      <abbr title="Young Living">YL</abbr> say
+      <q
+        >Learn how we're helping people experience real wellness and create
+        lasting abundance through the transformative power of essential oils.</q
+      >
```
Comment (marianzburlea) [inline web/simonamarcu2/html/primul/index.html:58]:
```<q
        >Learn how we're helping people experience real wellness and create
        lasting abundance through the transformative power of essential oils.</q
      >```

to
```
<q> 
  Learn how we're helping people experience real wellness and create
        lasting abundance through the transformative power of essential oils.
</q>
```

**Review by marianzburlea — APPROVED** [review]
💯

**Pull request merged**